packages/backend-defaults/api-report-alpha.md

```diff
@@ -0,0 +1,12 @@
+## API Report File for "@backstage/backend-defaults"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackendFeature } from '@backstage/backend-plugin-api';
+
+// @alpha
```

Comment:
Have we considered making this export a non alpha export given that the backend-defaults is not getting promoted to 1.x?

Reply:
That's a good question, the reason I kept it as alpha is that I'm not sure if this is a definitive solution for loading static or dynamic features or if it's just an experiment so far. What do you guys think @freben and @Rugvip?

Reply:
Let's do that imo, might even want to discuss whether this should be included by default since it's behind a config flag anyway.

Reply:
Done